server/data/curiousClient.ts

```diff
@@ -8,16 +8,16 @@ export default class CuriousClient {
   }
 
   async getLearnerProfile(prisonNumber: string, token: string): Promise<Array<LearnerProfile>> {
-    const learnerProfiles = (await CuriousClient.restClient(token).get({
+    const learnerProfiles = CuriousClient.restClient(token).get({
```

Comment:
Previously I dont think we were actually returning the promise, as we had the `await` keyword ... not even sure how it compiled or ran correctly at runtime to be honest because I'm not sure how the casting would have worked 🤷